cmd/horcrux/cmd/address.go

```diff
@@ -6,11 +6,12 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/strangelove-ventures/horcrux/pkg/pcosigner"
```

Comment:
nit: let's just call this `cosigner` instead of `pcosigner`

Reply:
Yes! 
Will fix it.